packages/terra-form-select/src/tag/Frame.jsx

```diff
@@ -722,21 +740,21 @@ class Frame extends React.Component {
         />
         {this.state.isOpen
           && (
-          <Dropdown
-            {...dropdownAttrs}
-            id={this.state.isOpen ? 'terra-select-dropdown' : undefined}
-            target={this.select}
-            isAbove={this.state.isAbove}
-            isTouchAccessible={isTouchAccessible}
-            isEnabled={this.state.isPositioned}
-            onResize={this.positionDropdown}
-            refCallback={(ref) => { this.dropdown = ref; }}
-            style={FrameUtil.dropdownStyle(dropdownAttrs, this.state)} // eslint-disable-line react/forbid-component-props
-          >
-            <Menu {...menuProps}>
-              {children}
-            </Menu>
-          </Dropdown>
+            <Dropdown
```

Comment:
nit: extra-space

Reply:
I think they were generated after auto-fix, linter didn't complain though. So I think its fine.

Reply:
I don't think changing the existing formatting unless it is required is not recommended either 🤔🧐😄

Reply:
As long as these changes are from the auto-formatting feature of VS Code using the terra configurations I think these changes are okay.

Any time I open these specific terra-form-select files my VSCode formatter reformats these code blocks. I have to turn off the plugin or manually unstage the changes block by block to prevent auto format updates.

Reply:
Good to know that. 🙂